shared-data/protocol/types/schemaV7/command/annotation.ts

```diff
@@ -0,0 +1,37 @@
+import type { CommonCommandRunTimeInfo, CommonCommandCreateInfo } from '.'
+
+export type AnnotationCreateCommand = CommentCreateCommand | CustomCreateCommand
+
+export type AnnotationRunTimeCommand =
+  | CommentRunTimeCommand
+  | CustomRunTimeCommand
+
+export interface CommentCreateCommand extends CommonCommandCreateInfo {
+  commandType: 'comment'
+  params: CommentParams
+}
+
+export interface CommentRunTimeCommand
+  extends CommonCommandRunTimeInfo,
+    CommentCreateCommand {
+  result: any
+}
+
+interface CommentParams {
+  message?: string
+}
```

Comment:
Does the `?` here indicate optionality? Should this be:

```suggestion
interface CommentParams {
  message: string
}
```